tests/test_output.py

```diff
@@ -20,7 +20,7 @@ def test_local_writer_json_stdout_default(self):
         output_writer = nextmv.LocalOutputWriter()
 
         with patch("sys.stdout", new=StringIO()) as mock_stdout:
-            output_writer.write(output)
+            output_writer.write(output, skip_stdout_reset=True)
```

Comment:
This wouldn't be necessary, if writing would reinstate it after it's done, right? :joy_cat: